src/components/TodoInput.js

```diff
@@ -7,6 +7,12 @@ class TodoInput extends Component {
   };
 
   onLabelChange = (event) => {
```

Comment:
```suggestion
  onLabelChange = ({ target }) => {
```
делай деструктуризацию